src/components/Player/GridControls.js

```diff
@@ -154,12 +155,65 @@ export default class GridControls extends Component {
     }
   };
 
+  _handleKeyDownVim = (key, shiftKey) => {
+    const actionKeys = {
+      ArrowLeft: 'left',
+      ArrowUp: 'up',
+      ArrowDown: 'down',
+      ArrowRight: 'right',
+      Backspace: 'backspace',
+      '{del}': 'backspace',
+      Delete: 'delete',
+      Tab: 'tab',
+      ' ': 'space',
+      '[': 'backward',
+      ']': 'forward',
+    };
+
+    const normalModeActionKeys = {
+      h: 'left',
+      j: 'down',
+      k: 'up',
+      l: 'right',
+      x: 'delete',
+      ' ': 'space',
+      '[': 'backward',
+      ']': 'forward',
```

Comment:
w -> selectNextClue(true), b -> selectNextClue(false) ? :D

Reply:
and maybe dd for delete word, though that's a bit unintuitive when it's vertical i guess

Reply:
i added w and b (good suggestions!) but haven't added dd yet since it seems like it would take some more work (need to add some queue state to check for the first d. I could take a stab at it in another PR along with some other ideas I had e.g. use `:32` to go to clue 32.

Reply:
ooh yea, `:32` would be sweet